client/src/components/ItemDetails/index.jsx

```diff
@@ -0,0 +1,382 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+// import icons
+import { IconContext } from 'react-icons';
+import {
+  MdKeyboardArrowUp,
+  MdKeyboardArrowDown,
+  MdAddShoppingCart,
+  MdPlaylistAdd,
+  MdDone,
+} from 'react-icons/md';
+import {
+  FaPinterest,
+  FaFacebookF,
+  FaTwitter,
+  FaCircleNotch,
+} from 'react-icons/fa';
+
+import InputField from '../common/InputField';
+
+import './ItemDetails.css';
+
+class ItemDetails extends Component {
+  state = {
+    cart: 'default',
+    wList: 'default',
+    color: '',
+    quantity: 0,
+    tags: [],
+  };
+
+  addToCartButton = {
+    adding: (
+      <React.Fragment>
```

Comment:
Should destructure this.
```suggestion
      <Fragment>
```

Reply:
I did it :+1: